lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -1594,6 +1594,30 @@ static Value createLinalgPayloadCalculationForElementwiseOp(
     Value result = convertScalarToDtype(b, loc, input, dtype);
     return result;
   }
+  if (isa<AtenAbsOp>(op))
+    return b.create<math::AbsOp>(loc, payloadArgs[0]);
+  if (auto reciprocal = dyn_cast<AtenReciprocalOp>(op)) {
+    if (!reciprocal.getType()
+            .cast<ValueTensorType>()
+            .getDtype()
+            .isa<mlir::FloatType>()) {
+      reciprocal.emitError("unimplemented: non-floating point dtype");
+      return nullptr;
+    }
+
+    Type elementType = payloadArgs[0].getType();
+    // runtime check
```

Comment:
nit: Comment need to be more informative to help with readability like: assert(element != 0)

Reply:
Thanks!